File: SpringBoot_404/demo/src/main/java/com/example/demo/product/domain/Product.java
package com.example.demo.product.domain;

import jakarta.persistence.*;
import lombok.Data;

@Data
@Entity
@Table(name="product")
public class Product {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long product_id;

    @Column(length = 30, nullable = false)
    private String product_name;

    @Column
    private int price;

    @Column
    private int total_quantity;

    @Column(length = 50, nullable = false)
    private String category;

    @Column(length = 100, nullable = false)
    private String content;

    @Column(length = 50, nullable = false)
    private String product_img;

    private String uuid;
}

File: React_404/cvs-app/src/productAdim/model/Product.js
export default class Product {
  constructor(
    product_id,
    product_name,
    price,
    total_quantity,
    category,
    content,
    product_img
  ) {
    this, (product_id = product_id);
    this.product_name = product_name;
    this.price = price;
    this.total_quantity = total_quantity;
    this.category = category;
    this.content = content;
    this.product_img = product_img;
  }
}
